examples/TCP_perf/service.cpp

```diff
@@ -113,15 +155,15 @@ void Service::ready()
   StackSampler::set_mode(StackSampler::MODE_DUMMY);
 #endif
 
-  static auto blob = net::tcp::construct_buffer(SIZE);
+  blob = net::tcp::construct_buffer(SIZE, '!');
 
 #ifdef USERSPACE_LINUX
   extern void create_network_device(int N, const char* route, const char* ip);
   create_network_device(0, "10.0.0.0/24", "10.0.0.1");
 #endif
 
   // Get the first IP stack configured from config.json
-  auto& inet = net::Interfaces::get(0);
+  auto& inet = net::Super_stack::get(0);
```

Comment:
No idea how this snuck in here. And how it even passed on jenkins? This also goes for the include line 20.